packages/astro/src/default/components/ResizablePanel.astro

```diff
@@ -23,18 +37,16 @@ if (!hasSidePanel) {
   min = '100%';
   max = '100%';
 }
+
+const panelClass = `overflow-hidden ${type === 'horizontal' ? 'h-full' : ''}`;
 ---
 
-<resizable-panel
-  class={classList ?? ''}
-  data-id={id}
-  data-type={type}
-  data-pos={pos}
-  data-min={min}
-  data-priority={priority}
-  data-max={max}
->
-  <div class=`container max-w-full ${type}` style=`--pos: ${pos}`>
+<resizable-panel class={className} data-id={id} data-type={type} data-pos={pos} data-min={min} data-max={max}>
+  <div
+    data-id="container"
+    class=`sm:grid relative w-full h-full max-w-full ${type === 'horizontal' ? 'sm:grid-cols-[var(--pos)_1fr]' : 'sm:grid-rows-[var(--pos)_1fr]'}`
```

Comment:
Why not use the `classNames()` method here?

Reply:
Oh good call! Will update it :+1:

Reply:
Yep, would be good to use `classNames` (I find it a lot easier to read personally).